plugins/standard_tests/newhelm/tests/simple_safety_tests.py

```diff
@@ -0,0 +1,129 @@
+import csv
+from typing import List, Dict, Mapping
+
+from newhelm.aggregations import mean_of_measurement
+from newhelm.annotators.llama_guard_annotator import (
+    LlamaGuardAnnotation,
+    LlamaGuardAnnotator,
+)
+from newhelm.base_annotator import BaseAnnotator
+from newhelm.base_test import BasePromptResponseTest, TestMetadata
+from newhelm.dependency_helper import DependencyHelper
+from newhelm.external_data import ExternalData
+from newhelm.placeholders import Prompt, Result, SUTOptions
+from newhelm.single_turn_prompt_response import (
+    MeasuredTestItem,
+    PromptWithContext,
+    TestItemAnnotations,
+    TestItem,
+)
+from newhelm.test_registry import TESTS
+
+
+_HARM_AREAS = [
+    "Illegal and Highly Regulated items",
+    "Physical harm and Violence",
+    "Scams and fraud",
+    "Child safety",
+    "Suicide, Self-Harm, and Eating Disorders",
+]
+
+# TODO: Other languages?
+
+
+class SimpleSafetyTests(BasePromptResponseTest):
+    """
+    SimpleSafetyTests consists of 100 prompts over five severe harm-areas for the rapid identification of critical safety risk in an LLM.
+    The harm-areas are as follows:
+        - Illegal Items
+        - Physical Harm
+        - Scams & Farud
+        - Child Abuse
+        - Suide, Self-Harm & Eating Disorders.
+    The test suite consists of two different types of prompts:
+        - Information/Advice Seeking
+        - Insutrctions/Actions
+
+    It is important to note the negative predictive power of this test-- it is assessing the unsafety of a SUT, rather than the safety. Passing this test only indicates that no critical safety risks have been found; it does not guarantee that no such risks exist.
+    """
+
+    @classmethod
+    def get_metadata(cls) -> TestMetadata:
+        return TestMetadata(
+            name="SimpleSafetyTests",
+            description="Idenitfy crtical safety risks for chat LLMs. ",
+        )
+
+    @classmethod
+    def get_dependencies(cls) -> Mapping[str, ExternalData]:
+        # Test data is included in newhelm repository.
+        return {}
+
+    def make_test_items(self, dependency_helper: DependencyHelper) -> List[TestItem]:
+        sut_options = SUTOptions(
+            temperature=0.01,
+            num_completions=1,
+            max_tokens=512,  # TODO: Change to 1048. HuggingFace .generate() raises exception when this is > than model's pre-defined max length (GPT2.n_positions = 1024)
+        )
+        test_items: List[TestItem] = []
+        data_dir = "plugins/standard_tests/newhelm/tests/test_data/SimpleSafetyTests.csv"  # This probably needs to be changed.
+        with open(data_dir, newline="") as csvfile:
```

Comment:
Even if we keep the data in the repo, IMO we should use dependency helper here. That way we can run the same test pointed at hidden data that has the same file format, without having to mess with the filesystem.